sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Platform/EnvironmentVariableConstants.cs

```diff
@@ -52,7 +52,9 @@ internal static class EnvironmentVariableConstants
         public const string WEBSITE_HOSTNAME = "WEBSITE_HOSTNAME";
 
         /// <summary>
-        /// INTERNAL ONLY. Used by Statsbeat to get the App Service Website Name.
+        /// INTERNAL ONLY.
+        /// Used by Statsbeat to get the App Service Website Name.
+        /// Used by LiveMetrics to detect running in Azure App Service.
```

Comment:
why does LiveMetrics need to detect this?

Reply:
2 reasons. 
- There is a flag on the LiveMetrics payload "IsWebApp" that indicates if an app is running in Azure AppService.
- If an application is deployed in Windows AppService, we'll use their self-reported Environment variable for process metrics.

Reply:
Can you elaborate on what special knowledge is required? 

Separately, there are many breaking changes between AI SDK and Distro, don't think we need to worry about. Most of the times, the breaking changes in OTel are for good only!